test/unittests/parser_test.cpp

```diff
@@ -594,10 +594,15 @@ TEST(parser, global_single_mutable_const_inited)
 
 TEST(parser, global_single_const_global_inited)
```

Comment:
Test name doesn't reflect the test anymore.

Reply:
Renamed